umbra/umbra.h

```diff
@@ -50,18 +50,23 @@ extern "C" {
  * that Umbra's actions occur at the right time.
  */
 #ifndef dr_get_tls_field
-# define dr_get_tls_field DO_NOT_USE_tls_field_USE_drmgr_tls_field_instead
-# define dr_set_tls_field DO_NOT_USE_tls_field_USE_drmgr_tls_field_instead
-# define dr_insert_read_tls_field DO_NOT_USE_tls_field_USE_drmgr_tls_field_instead
-# define dr_insert_write_tls_field DO_NOT_USE_tls_field_USE_drmgr_tls_field_instead
-# define dr_register_thread_init_event DO_NOT_USE_thread_event_USE_drmgr_events_instead
-# define dr_unregister_thread_init_event DO_NOT_USE_thread_event_USE_drmgr_events_instead
-# define dr_register_thread_exit_event DO_NOT_USE_thread_event_USE_drmgr_events_instead
-# define dr_unregister_thread_exit_event DO_NOT_USE_thread_event_USE_drmgr_events_instead
-# define dr_register_pre_syscall_event DO_NOT_USE_pre_syscall_USE_drmgr_events_instead
-# define dr_unregister_pre_syscall_event DO_NOT_USE_pre_syscall_USE_drmgr_events_instead
-# define dr_register_post_syscall_event DO_NOT_USE_post_syscall_USE_drmgr_events_instead
-# define dr_unregister_post_syscall_event DO_NOT_USE_post_syscall_USE_drmgr_events_instead
+#    define dr_get_tls_field DO_NOT_USE_tls_field_USE_drmgr_tls_field_instead
```

Comment:
Probably best to leave big changes to unrelated code to a separate commit: it will make it hard to see what this PR itself is changing.

Reply:
Can I revert back with bad formatting and then, immediately after, do a separate PR with proper formatting on umbra files that is done with clang-format please?

Reply:
I think the indent/format issues in this PR were all in the new test and not in umbra.h or umbra_32.c?  Or if there were any in those files they were just one or two which could be manually fixed.  Seems like this PR can go in w/o adding bad formatting (or at least that I noticed in my review)?  Afterward, sure, if you want to clang-format the umbra files and set up a CI check: that's part of #2107.